### DISCUSSION TITLE
Indicazione età dei soggetti

### DISCUSSION THREAD

**Discussion by siscomanpr** [italia/ansc#1199] [Miglioramenti] [answered]
Alcuni nostri clienti ci segnalano che spesso capita che ricevano atti esteri da trascrivere dove al posto della data di nascita del soggetto, è indicata solo l'età; nello specifico compare per esempio: "di anni 30". 
Negli estratti ovviamente dovrebbe poi comparire questa stessa dicitura al posto della data di nascita del soggetto.

Il sistema ANSC ad oggi non ci risulta gestisca questa informazione. E' prevista un'implementazione in tal senso? 
Un comune attualmente è bloccato con la trascrizione di alcuni atti non essendo a conoscenza della data di nascita precisa.

Restiamo in attesa di un riscontro urgente.

Cordiali saluti
Siscom SPA

**Comment by acolosimosogei** [comment]
Buongiorno,

proporremo agli esperti ANUSCA questa discussion, la avviseremo quanto prima del loro riscontro.

Grazie mille e buona giornata,
Assistenza ANSC

**Comment by acolosimosogei** [comment]
Buonasera,

il problema è stato proposto ad ANUSCA che ci ha confermato di intervenire, la avviseremo quando il problema riportato sarà stato lavorato.

Grazie,
Assistenza ANSC

**Comment by acolosimosogei** [comment, marked as answer]
Salve, 

l'intervento è previsto da piano operativo per fine settembre 2025.

Assistenza ANSC

**Comment by asqueglia** [comment]
Buon pomeriggio, abbiamo creato la relativa issue che potete consultare per monitorare il rilascio dell'intervento #1400 chiudiamo pertanto la discussion.
